android/src/main/java/com/reactlibrary/RNSpotifyRemoteAuthModule.java

```diff
@@ -79,6 +94,14 @@ public void onActivityResult(Activity activity, int requestCode, int resultCode,
                     if (authPromise != null) {
                         String token = response.getAccessToken();
                         mAuthResponse = response;
+                        authPromise.resolve(Convert.toMap(response, 'TOKEN'));
+                    }
+                    break;
+
+                    ase CODE:
```

Comment:
Hi, I haven't tested the code but reviewing it seems that 'c' is missing

```suggestion
                    case CODE:
```

Reply:
Oh, sorry I changed it first in my app code then copied the changes to the forked code and missed this 😅